src/components/articleTeaser/index.stories.mdx

```diff
@@ -0,0 +1,54 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { Box } from '@chakra-ui/react';
+
+import ArticleTeaser from './index.tsx';
+
+<Meta
+  title="Patterns/Data display/Article Teaser"
+  component={ArticleTeaser}
+  args={{
+    nativeImageSize: '320x320',
+    title: 'Placeholder.com',
+    text: 'A very cool service that provides placeholder images. You can learn more about the options and usage here.',
+    url: 'https://placeholder.com',
+    maxImgW: '4xl',
+  }}
+  argTypes={{
+    maxImgW: {
+      options: ['xl', '2xl', '3xl', '4xl', '5xl', '6xl'],
+      control: 'select',
+    },
+  }}
+/>
+
+export const Template = ({ nativeImageSize, ...args }) => (
+  <Box maxW={{ xs: '100%', lg: '600px' }}>
+    <ArticleTeaser
+      {...args}
+      imageUrl={`https://via.placeholder.com/${nativeImageSize}`}
+    />
+  </Box>
+);
+
+# ArticleTeaser
+
+## Overview
+
+<Canvas>
+  <Story name="Overview">{Template.bind({})}</Story>
+</Canvas>
+
+<ArgsTable story="Overview" />
+
+## Responsive
+
+Bigger image on mobile viewport.
+
+<Canvas>
+  <Story name="Responsive" args={{ maxImgW: { sm: 'full', md: '3xl' } }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+<ArgsTable story="Responsive" />
```

Comment:
Do we want to add this on our stories or do we want to use better the responsive tool in storybook for checking the behaviour in the different breakpoints?

I was thinking only show the variations as stories but no the responsive ones. Otherwise, we will have so many stories. 
What do you think?

Reply:
We already can check the behaviour on different breakpoints (the viewports addon allows us to do so). I'm showcasing something different here. The `maxImgW` prop accepts `ResponsiveValue` so I want to show that one can set different image sizes on mobile on desktop.
Are there stories where such a showcase is missing? I tend to add them when I add a component with responsive props, but if we miss that somewhere then it's a good idea to add it.

Reply:
This will be the first one after the discussion we have regrading the image flexibility. We need to do the same for the sections (the one already merged).

Reply:
Ah yes, that's something we'd want to add to the sections too. Shall I do that or do we tackle that in a separate PR?

Reply:
Mmmm good question. I will say better in a separate PR.